src/game.ts

```diff
@@ -88,6 +90,24 @@ export class Game {
 		}
 	}
 
+	/**
+   * Get or Set custom game tag
+   */
+	tag(tagName: string, value?: string): string | undefined {
```

Comment:
I don't want a generic key-value tag for the reasons explained in #33.

Reply:
Continuing discussion in #33